### ISSUE TITLE
Some bluetooth devices are no longer showing up after upgrade to 0.2.0

### ISSUE THREAD

**Issue by xanderdeseyn** [dotintent/MultiPlatformBleAdapter#102]
We are using this library through react-native-ble-plx. We noticed that since MultiplatformBleAdapter was upgraded, we can no longer detect our bluetooth beacons. They are very simple bluetooth devices without any characteristics or services. Is it possible that these properties break the library?

**Comment by mmilvydas112** [comment]
yes, it relates to these changes https://github.com/dotintent/MultiPlatformBleAdapter/commit/14bd1f9fb3a3ff809b20bf8763821ce244dbe361

Did not investigate further, played around with different versions with one commit before this one, but wasn't able to make it work. Anyone know compatibility fix?
